css/styles.css

```diff
@@ -158,3 +158,598 @@ html {
+  line-height: 24px;
+  color: #fff;
+  margin: 4px 7px;
+}
+
+.text-box-buttons {
+  margin: auto 7px;
+  width: 100%;
+  margin-bottom: 4px;
+}
+
+.button-3 {
+  padding: 10px 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  color: #fff;
+  margin-right: 12px;
+  background: rgba(255, 255, 255, 0.24);
+  cursor: pointer;
+}
+
+.button-3:hover {
+  background: #fff;
+  color: #091e42;
+}
+
+.button-4 {
+  position: relative;
+  height: 48px;
+  width: 100%;
+  bottom: 0;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  outline: none;
+  border: 1px solid #ff6b00;
+}
+
+.button-4:hover {
+  background: #fcad75;
+  color: #212121;
+}
+
+/* About-me section css start from here */
+
+.about-me {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  row-gap: 130px;
+  width: 412px;
+  height: 1850px;
+  background: url(/images/aboutme.png);
+}
+
+.about-me-container {
+  display: flex;
+  position: relative;
+  top: 134px;
+  flex-direction: column;
+
+  /* border: 2px solid purple; */
+  width: 327px;
+  height: 386px;
+  margin: auto;
+}
+
+.about-me-heading h1 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.about-me-paragraph p {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 100;
+  font-size: 15px;
+  margin: 9px auto;
+  line-height: 24px;
+  color: #e1c9c9;
+}
+
+/* Language-container-section start from here */
+
+.language-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  border-top: 1px solid #fff;
+  width: 327px;
+  height: 1186px;
+  margin: auto;
+}
+
+/* Language heading section start from here */
+
+.language-heading {
+  display: flex;
+  flex-direction: column;
+
+  /* border: 2px solid purple; */
+  width: 225.46px;
+  height: 281.46px;
+  margin-top: 60px;
+}
+
+.language-heading ul {
+  list-style: none;
+  width: 100%;
+}
+
+.language-heading ul li {
+  display: flex;
+  align-items: center;
+  font-family: 'Meera Inimai', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 30px;
+}
+
+.language-heading ul li a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.language-heading ul li::before {
+  /* content: '\20DF'; */
+  content: '\25C6';
+  font-size: 50px;
+  margin-right: 17px;
+  color: #ff6b00;
+
+  /* background-color: #FF6B00; */
+}
+
+.language-list-items {
+  display: flex;
+  flex-direction: column;
+  width: 186px;
+  height: 216px;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  list-style: none;
+}
+
+.language-list-items ul li {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
```

Comment:
According to Figma design, This`font-size` is 22px
![image](https://user-images.githubusercontent.com/61361037/212377584-8f9f6cf1-37cf-4b29-b75b-0b6e275d419f.png)